pilot/pkg/bootstrap/server.go

```diff
@@ -994,20 +989,9 @@ func getDNSNames(args *PilotArgs, host string) []string {
 	if args.Revision != "" && args.Revision != "default" {
 		knownHosts = append(knownHosts, "istiod"+"-"+args.Revision)
 	}
-
-	for _, altName := range knownHosts {
-		name := fmt.Sprintf("%v.%v.svc", altName, args.Namespace)
```

Comment:
aren't we losing this? will break all clients?

Reply:
Actually surprised the tests pass... maybe I misunderstand

Reply:
you are right